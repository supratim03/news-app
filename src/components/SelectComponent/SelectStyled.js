import styled from "styled-components";

export const SelectBlock = styled.select`
    height: 35px;
    width: 175px;
    border-bottom: 1px solid;
    border-top: none;
    border-left: none;
    border-right: none;
`;