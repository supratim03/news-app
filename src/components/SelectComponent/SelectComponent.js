import React from "react";
import { SelectBlock } from "./SelectStyled";

const SelectComponent = () => {
    return (
        <SelectBlock>
            <option value="newest">Newest First</option>
            <option value="oldest">Oldest First</option>
            <option value="relevance">Popular First</option>
        </SelectBlock>
    )
}

export default SelectComponent;