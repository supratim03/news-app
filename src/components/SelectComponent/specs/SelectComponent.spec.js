import React from "react";
import { shallow } from "enzyme";
import SelectComponent from "../SelectComponent";

describe("ArticleDetail component", () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(
            <SelectComponent />
        );
    });

    it("renders correctly", () => {
        expect(wrapper).toMatchSnapshot();
    });
});