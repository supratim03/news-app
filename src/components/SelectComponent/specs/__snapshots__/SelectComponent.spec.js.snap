// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleDetail component renders correctly 1`] = `ShallowWrapper {}`;
